lesson3/fibo.py

```diff
@@ -0,0 +1,23 @@
+import sys
+
+
+def fibo_easy(n):
```

Comment:
These look fine, not sure why you would parse the args here. Again, you don't really use arg parsing that often in python and when you do you don't do it manually, but you use argparse instead. Anyway the goal of the task is to show the students how you can juggle with functions since they are first class citizens in python, thus I would try to keep it as simple as possible.